src/chain-wallet-libs/bindings/wallet-core/src/wallet.rs

```diff
@@ -100,6 +102,53 @@ impl Wallet {
             .map_err(|_| Error::invalid_spending_counters())
     }
 
+    fn sign_transaction_impl<P: Payload>(
+        &mut self,
+        settings: &Settings,
+        valid_until: BlockDate,
+        lane: u8,
+        payload: P,
+        auth: P::Auth,
+    ) -> Result<Transaction<P>, Error> {
+        let mut builder = wallet::TransactionBuilder::new(&settings, payload, valid_until);
+
+        let value = builder.estimate_fee_with(1, 0);
```

Comment:
Hm.. why we are hardcoding this?

Reply:
Good, catch I have taken it from the previous implementation, will update it